prac_05/emails.py

```diff
@@ -0,0 +1,22 @@
+""""
+estimate: 30 mins
+actual: 40 mins
+"""
+email_name = {}
+
```

Comment:
You could just use while email is not empty instead and there would no be any need to have break